README.md

```diff
@@ -202,6 +202,20 @@ Jitpack makes builds of each commit on its own servers. You can view the results
 
 https://jitpack.io/com/github/heroiclabs/nakama-java/<_commit_>/build.log
 
+
+## Generate Docs
+
+API docs are generated with javadoc and deployed to GitHub pages.
+
+When changing the API comments, rerun javadoc and commit the changes in `docs/*`.
+
+To run javadoc:
+
+```
+cd nakama-java/src/main/com/heroiclabs/nakama
```

Comment:
this should be `src/main/java/com/heroiclabs/nakama` (notice missing `java`)

Reply:
As a matter of fact - these two lines need to be consolidated into this:

---

In the root of the repo, run the following command:

```
./gradlew javadoc
```

---